Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:7.9.0-alpine
+MAINTAINER Lukasz Gornicki <derberg@wp.pl>
+# add bash and git
+RUN apk update && apk add --no-cache bash git openssh && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /var/cache/apk/*
+# clone and install skeleton
+RUN git clone https://github.com/YaaS/docpad-skeleton-apidocs.git
+WORKDIR /docpad-skeleton-apidocs
+# install dependencies of the skeleton
+RUN npm run prepare
+# initialize your skeleton with sample data
+RUN npm run init
+# volume name
+VOLUME /docpad-skeleton-apidocs
+EXPOSE 9778
```

Comment:
you can add default command to execute after start container:
```docker
CMD ["/bin/bash"]
```
thanks to this you can run container without last parameter:
```bash
docker run -it --rm --name api-doc-sample -p 9778:9778 derberg/dsa-quickstart
```